src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ActionPlanController.kt

```diff
@@ -64,21 +62,11 @@ class ActionPlanController(
   @ResponseStatus(HttpStatus.OK)
   @PreAuthorize(HAS_VIEW_ACTIONPLANS)
   fun getActionPlan(@PathVariable @Pattern(regexp = PRISON_NUMBER_FORMAT) prisonNumber: String): ActionPlanResponse {
-    val response = actionPlanMapper.fromDomainToModel(actionPlanService.getActionPlan(prisonNumber))
-
-    // Map each goal response to its corresponding version with notes
```

Comment:
Mapping the notes into the goal is now all done consistently in one place (in the goalMapper, via the actionPlanMapper in this case); rather than it being dotted around in several places in the codebase, and essentially being done by mutating the goalResource (strictly speaking it was a copy operation on line 81, but for all intents and purposes we mutating it)